lib/mockbuilder.py

```diff
@@ -149,7 +149,8 @@ def _install_external_dependencies(self, package):
 
     def _create_build_directory(self, package):
         self.build_dir = os.path.join(
-            os.getcwd(), 'build', self.timestamp, package.name)
+            os.path.abspath(CONF.get('default').get('work_dir')), 'build',
```

Comment:
Currently this will be build/build, this does not make sense. I suggest we use mock_build